osconfig_tests/test_suites/package_management/package_management.go

```diff
@@ -17,7 +17,10 @@ package packagemanagement
 import (
 	"context"
 	"fmt"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/config"
+	"github.com/GoogleCloudPlatform/compute-image-tools/osconfig_tests/gcp_clients"
```

Comment:
fix imports

Reply:
my intellij does not do autoimport, need to configure it properly. my vim does that though